nlbsg/types.py

```diff
@@ -236,7 +237,8 @@ class Item:
         cluster_name=None,
         category_name=None,
         collection_code=None,
-        collection_min_age_limit=None
+        collection_min_age_limit=None,
+        not_on_loan=False,
```

Comment:
How about having both `on_loan` and `reserved` instead of `not_on_loan`?

Reply:
Sorry, ignore my previous comment. How about just `available`? I think having a negative boolean variable may be confusing, for example:

```
if not not_on_loan:
    # do stuff
```

Reply:
Okay! you're the boss :)